### PR TITLE
#107 Support logging should refactor configuration reading logic

### PR DESCRIPTION
**Pull request by yanghua**
Signed-off-by: yanghua <yanghua1127@gmail.com>

cc @drasko , please review this PR , thanks!

### CODE REVIEW

File: support/logging/server.go [hunk 1]
```diff
@@ -223,28 +219,28 @@ func httpServer() http.Handler {
...
 }
 
-func getPersistence(config Config) persistence {
+func getPersistence() persistence {
 	var retValue persistence
-	if config.Persistence == PersistenceFile {
-		retValue = &fileLog{filename: config.LogFilename}
-	} else if config.Persistence == PersistenceMongo {
-		ms, err := connectToMongo(&config)
+	if configuration.Persistence == PersistenceFile {
```
Comment (drasko) [inline support/logging/server.go]: `switch` would be more readable construct here

**Comment by yanghua** [comment]
cc @drasko

**Comment by jduranf** [comment]
Maybe is out of scope of this PR, but parameters like Hostname or MongoURL shouldn't be ServiceAddress or MongoDBHost? To use the same names used in core services...

**Comment by yanghua** [comment]
@jduranf you are right, I will create a new issue about refactoring the same semantic configuration item with the unified key. However this PR just refactored the way of reading the configuration. They are two things. Thanks!

**Pull request merged**